rules/windows/file/file_event/file_event_win_create_susp_evtx.yml

```diff
@@ -0,0 +1,28 @@
+title: Create Suspicious EVTX
+id: 65236EC7-ACE0-4F0C-82FD-737B04FD4DCB
+status: experimental
+description: Detects create new suspicious evtx files.
+references:
+    - not found
+author: D3F7A5105
+date: 2022/12/31
+logsource:
+    category: file_event
+    product: windows
+    definition: Check your sysmon configuration for monitoring Eventid 11. Added <TargetFilename condition="end with">.evtx<TargetFilename>
+detection:
+    selection:
+        TargetFilename|endswith: '.evtx'
+    filter1:
+        TargetFilename|contains: 'Windows\System32\WinEVT\Logs\'
+    filter2:
+        Image: "C:\\Windows\\Explorer.EXE"
+    filter3:
+        Image: "C:\\Windows\\system32\\DllHost.exe"
+    condition: selection and not filter1 and not filter2 and not filter3
+falsepositives:
+    - Admin activity
+level: high
+tags:
+    - attack.defense_evasion
+    - attack.t1562.002
```

Comment:
Can you give an example of a suspicious EVTX file? Or describe how creating such an EVTX file can be suspicious.

Reply:
an attacker can change the event storage path to a different path than the one defined by the administrator. in this case, the log will be created in a different directory, or in the same but with a different name.